romanesco/plugins/vtk/converters/geometry/validate_vtkpolydata_serialized.json

```diff
@@ -2,5 +2,6 @@
     "inputs": [{"name": "input", "type": "geometry", "format": "vtkpolydata.serialized"}],
     "outputs": [{"name": "output", "type": "boolean", "format": "boolean"}],
     "script": "output = isinstance(input, (str, unicode))",
+    "extensions": ["vtkpolydata.serialized", "vtk"],
```

Comment:
I think these should just be "vtkpolydata", "vtktable", "vtktree" for the first extension for VTK types. I agree that "vtk" should also be included. I think we should make all extensions not have dots in them.